samples/CoreAnimationTest/CoreAnimationTest/StoryBoardViewController.m

```diff
@@ -0,0 +1,32 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "StoryBoardViewController.h"
+
+@implementation StoryBoardViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    // Main storyboard view
+    //UIStoryboard* xamlStoryBoard = [UIStoryboard storyboardWithName:@"CoreAnimationTest" bundle:[NSBundle mainBundle]];
+
+    // CALayer
+    //UIViewController* caLayerVC = [xamlStoryBoard instantiateViewControllerWithIdentifier:@"CALayerSB"];
+    //[self addMenuItemViewController:caLayerVC andTitle:@"CALayerSB"];
```

Comment:
It's a good practice to exercise storyboards and the full Xib2Nib import flow. I didn't have time to include this test in this sprint though.

---
In reply to: [84973785](https://github.com/Microsoft/WinObjC/pull/1204#discussion_r84973785) [](ancestors = 84973785)